src/components/molecules/search-results-list/search-results-list.jsx

```diff
@@ -57,11 +57,23 @@ const SearchResultsList = ({
 		</select>
 	);
 
+	/**
+	 *	Computes index value of result item taking current items per page (pageunit) into consideration
+	 *
+	 *	@param {number} Result index in current response
+	 *	@returns {number} Computed result index number
+	 */
+	const getResultIndex = (index) => {
+		const page = urlQuery.get('pageunit') || 0;
```

Comment:
Once again, why is this here? Is this different from `resultsPerPage`??

Reply:
Updated as discussed. Using `resultsPerPage` and `currentPage`.